src/languages/vbnet.js

```diff
@@ -1,9 +1,9 @@
-/**
- * Language: Visual Basic .NET
- * Description: Visual Basic .NET (VB.NET) is a multi-paradigm, object-oriented programming language, implemented on the .NET Framework.
- * Authors: Poren Chiang <ren.chiang@gmail.com>, Jan Pilzer
- * Website: https://docs.microsoft.com/dotnet/visual-basic/getting-started/
- */
+/*
+Language: Visual Basic .NET
+Description: Visual Basic .NET (VB.NET) is a multi-paradigm, object-oriented programming language, implemented on the .NET Framework.
+Authors: Poren Chiang <ren.chiang@gmail.com>, Jan Pilzer
+Website: https://docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/
```

Comment:
FYI: I had no issues with this URL change if it was linking to a more general page...

Only issue was with the stylistic * stuff. :)